### PR TITLE
3214 - Add ui level badge + Ui Form improvements 

### PR DESCRIPTION
**Pull request by minotogna**
https://github.com/openforis/fra-platform/assets/1138156/68811af6-ee17-4bf3-a110-e59f31e73cdf

### CODE REVIEW

**Comment by sorja** [comment]
should the badge be on the left?

File: src/client/pages/CountryHome/Repository/Item/Item.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
+@import 'src/client/style/partials';
+
+$color-private: lighten($ui-destructive, 2.5%);
+$color-public: darken($ui-status-accepted, 15%);
```
Comment (sorja) [inline src/client/pages/CountryHome/Repository/Item/Item.scss]: should we use less bright colors? or inverse (white or transparent bg and border)

Comment (minotogna) [reply src/client/pages/CountryHome/Repository/Item/Item.scss]: I can try later . I really want to close this @sorja

File: src/client/pages/CountryHome/Repository/Item/Item.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  const { repositoryItem } = props
+
+  const { t } = useTranslation()
+
+  const level = repositoryItem.props.public ? 'public' : 'private'
+
+  return (
+    <div className="repository-item">
+      <RepositoryLink datum={repositoryItem} />
+      <div className={classNames('repository-item__badge', level)}>{t(`common.${level}`)}</div>
```
Comment (sorja) [inline src/client/pages/CountryHome/Repository/Item/Item.tsx:25]: should we introduce common component Badge?

`<Badge level={BageLevels.public} label='label' />`

Comment (minotogna) [reply src/client/pages/CountryHome/Repository/Item/Item.tsx:25]: Not now @sorja .
I really want to close this repository

**Review by sorja — APPROVED** [review]

**Pull request merged**